libs/mrwebrtc-unityplugin/src/pch.h

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#ifdef _DEBUG
+#define DBG_NEW new (_NORMAL_BLOCK, __FILE__, __LINE__)
+#else
+#define DBG_NEW new
+#endif
+
+#include <cassert>
+#include <cstdint>
+#include <functional>
+#include <string>
+#include <string_view>
+
+#if defined(MR_SHARING_WIN)
+
+#include "targetver.h"
+
+#define WEBRTC_WIN 1
+
+#ifndef WIN32_LEAN_AND_MEAN
+#define WIN32_LEAN_AND_MEAN
+#endif
+
+#ifndef NOMINMAX
+#define NOMINMAX
+#endif
+
+#include <windows.h>
+
+#elif defined(MR_SHARING_ANDROID)
+
+#define WEBRTC_POSIX 1
+#define WEBRTC_ANDROID 1
+
+#endif
```

Comment:
```suggestion
#if defined(MR_UNITYPLUGIN_SHARING_WIN)

#include "targetver.h"

#define WEBRTC_WIN 1

#ifndef WIN32_LEAN_AND_MEAN
#define WIN32_LEAN_AND_MEAN
#endif

#ifndef NOMINMAX
#define NOMINMAX
#endif

#include <windows.h>

#elif defined(MR_UNITYPLUGIN_SHARING_ANDROID)

#define WEBRTC_POSIX 1
#define WEBRTC_ANDROID 1

#endif
```
I am not sure you need the `#define WEBRTC_*` lines since this project is not including WebRTC code directly AFAICS (but they shouldn't hurt so fine to leave them).